Lesson30/Firat/my-app/src/components/Exercise2/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+
+export const Exercise2 = () => {
+  const [label, setLabel] = useState("On");
```

Comment:
The code in this component would not really work. Instead, change it like this:
For the state value, use boolean (true/false).

Then, you can easily update it in the setLabel function the way you do already.

Later, you have pattern matching: if label is true show "on", otherwise "off"